components/esm/defs.hpp

```diff
@@ -54,6 +54,26 @@ struct Position
     {
         return osg::Vec3f(pos[0], pos[1], pos[2]);
     }
+
+    bool inline operator== (const Position& position) const
```

Comment:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c86-make--symmetric-with-respect-to-operand-types-and-noexcept

Reply:
Just for reference: there is a lot of asymmetric comparison operators in OpenMW.